elixir/apps/api/lib/api/controllers/fallback_controller.ex

```diff
@@ -0,0 +1,31 @@
+defmodule API.FallbackController do
+  use Phoenix.Controller
+
+  def call(conn, {:error, :not_found}) do
+    conn
+    |> put_status(:not_found)
+    |> put_view(json: API.ErrorJSON)
+    |> render(:"404")
+  end
+
+  def call(conn, {:error, :unauthorized}) do
+    conn
+    |> put_status(403)
+    |> put_view(json: API.ErrorJSON)
+    |> render(:"403")
```

Comment:
Just curious if you meant `401` here?

```suggestion
    |> put_status(401)
    |> put_view(json: API.ErrorJSON)
    |> render(:"401")
```

Reply:
Yep, I always mix this up with HTTP statuses.  I always forget that `Unauthorized` in HTTP is really `Unauthenticated` and `Forbidden` is really `Unauthorized`.

Reply:
@bmanifold same here 🤣